src/components/searchBarV2/accountsAndDeletegates.css

```diff
@@ -0,0 +1,112 @@
+@import '../app/variablesV2.css';
+
+.wrapper {
+  width: 100%;
+  height: auto;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  margin-top: 20px;
+}
+
+.header {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+
+  & > label {
+    font-family: var(--content-font);
+    font-size: var(--font-size-small-secondary);
+  }
+}
+
+.subTitles {
+  width: 100%;
+  height: auto;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+
+  & > label {
+    font-family: var(--content-font);
+    font-size: var(--font-size-small-secondary);
+    color: var(--color-text-blue-gray);
+  }
+}
+
+.content {
+  width: 100%;
+  height: auto;
+  max-height: 184px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+
+  & .accountRow {
+    width: 100%;
+    height: auto;
+    padding: 5px 0;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    cursor: pointer;
+  }
+
+  & .accountInformation {
+    height: 36px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    margin-left: 20px;
+
+    & > div {
+      display: flex;
+      justify-content: center;
+      align-items: flex-start;
+    }
+  }
+
+  & .tag {
+    background: var(--color-text-slate-gray);
+    color: var(--color-white);
+    font-size: var(--footnote-font-size);
+    font-weight: var(--font-weight-bold);
+    padding: 3px;
+    box-sizing: border-box;
+    margin-left: 7px;
+    border-radius: 5px;
+  }
+
+  & .accountTitle {
+    font-family: var(--content-font);
+    font-size: var(--paragraph-font-size-l);
+    color: var(--color-text-mastricht-blue);
+    font-weight: var(--font-weight-bold);
```

Comment:
use mixins

Reply:
done

Reply:
Should have the font-weigh, also should be using the `heading` mixin no the `content`